src/Behat/Behat/Output/Node/Printer/Html/templates/navigator.html

```diff
@@ -0,0 +1,20 @@
+<nav class="navbar navbar-inverse navbar-fixed-top">
+    <div class="container">
+        <div class="navbar-header">
+            133 scenarios (133 passed, 0 Failed, 0 undefined, 0 pending, 0 Skipped)
+            <br>
+            679 steps (679 passed, 0 Failed, 0 undefined, 0 pending, 0 Skipped)
+            <br>
+            1m31.73s (memory)
```

Comment:
don't know how lost the track on this :( , replaced values with their template's variables